scale-tests/README.md

```diff
@@ -42,29 +42,95 @@ This will upload the "tests assembly" JAR to S3 and output its public object URL
 
 ## Running Tests
 
-### Batch
-[To be filled in]
-
-### Streaming
-
 Cluster requirements:
 - Enough resources to run the workload given the parameters.
 - Package repository containing a version of Spark with quotas.
 
-#### 0. Getting a shell session that's able to run the scripts
+### Getting a shell session that's able to run the scripts
 
 ```bash
 your-machine $ git clone git@github.com:mesosphere/spark-build.git
 your-machine $ cd spark-build
-your-machine $ docker run -it --net=host -v (pwd):/spark-build -v </path/to/soak/key>:/ssh/key mesosphere/spark-build:latest
+your-machine $ docker run -it --net=host -v $(pwd):/spark-build -v </path/to/soak/key>:/ssh/key mesosphere/spark-build:latest
 docker-container # cd /spark-build
 docker-container # pip3 install -r tests/requirements.txt
 docker-container # export PYTHONPATH=$(pwd)/testing:$(pwd)/spark-testing
 docker-container # dcos cluster setup --insecure --username=<username> --password=<password> <cluster-url>
 docker-container # eval "$(ssh-agent)"
+docker-container # chmod 600 ccm.pem
```

Comment:
Does this makes sense from the context of the docker container? The file `ccm.pem` does not exist, and is added as `/ssh/key`. One could change the mount argument in the docker run command to `-v </path/to/soak/key>:/ssh/key:ro` to indicate that this should be readonly.